src/app/home/routing.tsx

```diff
@@ -10,7 +10,7 @@ const Stack = createStackNavigator<HomeParamList>();
 
 export const HomeStackComponent: React.FC = () => {
     return (
-        <Stack.Navigator headerMode={'none'}>
+        <Stack.Navigator headerMode={'none'} initialRouteName={'HomeAlbumsScreen'}>
```

Comment:
Remove this before pushing final commit, we don't want to see this in develop branch